README.md

```diff
@@ -53,46 +53,44 @@ Other tasks:
 ```
 
 ```
->   Usage : steppe [-d startdir] [-f files.in] [-q] [-s] [-e] [-o] [-g]
+>   Usage : steppe [-d startdir] [-f files.in] [-q] [-e] [-o] [-g]
 >      -d : supply working directory (default=.)
 >      -f : supply list of input files (default=files.in)
 >      -q : quiet mode, don't print message to check logfile.
 >      -p : prints progress bar
->      -s : use SOILWAT model for resource partitioning.
 >      -e : echo initialization results to logfile
 >      -o : write SOILWAT output to output files. Contains average over all iterations and standard deviation.
 >      -g : use gridded mode
 >      -i : write SOILWAT output to output files for each iteration
 >-STdebug : generate sqlite database with STEPWAT information
 ```
 
-* Run the gridded version (-g) of STEPWAT2 using SOILWAT2 to drive the water cycle (-s):
+* Run the gridded version (-g) of STEPWAT2:
 
 ```
 cd testing.sagebrush.master/
-./stepwat -f files.in -g -s
+./stepwat -f files.in -g
 ```
 
 
-* Run the non-gridded version of STEPWAT2 from the Stepwat_Inputs/ folder using SOILWAT2
-  to drive the water cycle (-s):
+* Run the non-gridded version of STEPWAT2 from the Stepwat_Inputs/ folder:
```

Comment:
I think we still need to retain reference to SOILWAT2 here. The default now is that we are always using SOILWAT2. The originial comment can be retained, but the `(-s)` can be deleted.